test/test_train_imagenet.py

```diff
@@ -146,8 +147,10 @@ def train_loop_fn(model, loader, device, context):
       xm.optimizer_step(optimizer)
       tracker.add(FLAGS.batch_size)
       if x % FLAGS.log_steps == 0:
-        print('[{}]({}) Loss={:.5f} Rate={:.2f}'.format(device, x, loss.item(),
-                                                        tracker.rate()))
+        print('[{}]({}) Loss={:.5f} Rate={:.2f}, GlobalRate={:.2f}, Time={}'
```

Comment:
This will need, for uniformity, be done in MNIST and CIFAR as well.
So maybe a utility API in test_utils.py?